### PR TITLE
Make lots of responsive improvements to the schedule page

### PR DESCRIPTION
**Pull request by lboynton**
- Hide avatars and talk abstracts on mobile
- Hide talk abstracts on tablet
- Show everything on larger than tablet
- Avatar/abstract still available by clicking on the speaker's name

### CODE REVIEW

File: schedule.html [hunk 1]
```diff
@@ -109,7 +118,7 @@
                 <td>
                     <strong>Break &amp; refreshments</strong>
                     <p>Tea and coffee will be served.</p>
-                    <p>Free water will be available throughout the day. Soft drinks will also be available to purchase (cash only).</p>
+                    <p>Free water and soft drinks available to purchase (cash only).</p>
```
Comment (asgrim) [inline schedule.html:121]: Free water available to purchase? :P

Comment (lboynton) [inline schedule.html:121]: Would an Oxford comma help clarify that sentence?

**Comment by lboynton** [comment]
Damn, was going to add that comma today

**Comment by asgrim** [comment]
Can still do it ;)

**Comment by lboynton** [comment]
I'll add it to Trello...

**Pull request merged**